packages/ui/Button/Button.js

```diff
@@ -177,7 +177,9 @@ const ButtonBase = kind({
 		return (
 			<div role="button" {...rest} aria-disabled={disabled} disabled={disabled}>
 				<div className={css.bg} />
-				<div className={css.client}>{icon}{children}</div>
+				<div className={css.clientContainer}>
```

Comment:
It was my expectation that this change would not necessitate adding any new DOM.